docs/root/intro/arch_overview/load_balancing.rst

```diff
@@ -247,6 +227,72 @@ To sum this up in pseudo algorithms:
   total_health = min(100, Σ(health(P_0)...health(P_X))
   load to P_X = 100 - Σ(percent_load(P_0)..percent_load(P_X-1))
 
+.. _arch_overview_load_balancing_panic_threshold:
+
+Panic threshold
+---------------
+
+During load balancing, Envoy will generally only consider healthy hosts in an upstream cluster.
+However, if the percentage of healthy hosts in the cluster becomes too low, Envoy will disregard
+health status and balance amongst all hosts. This is known as the *panic threshold*. The default
+panic threshold is 50%. This is :ref:`configurable <config_cluster_manager_cluster_runtime>` via
+runtime as well as in the :ref:`cluster configuration
+<envoy_api_field_Cluster.CommonLbConfig.healthy_panic_threshold>`. The panic threshold
+is used to avoid a situation in which host failures cascade throughout the cluster as load
+increases.
+
+Panic thresholds work in conjunction with priorities. If number of healthy hosts in given priority
+goes down, Envoy will try try shift some traffic to lower priorities. If it succeeds finding enough 
+healthy hosts in lower priorities, Envoy will disregard panic thresholds. In mathematical terms, 
+if total health across all priority levels is 100%, Envoy disregards panic thresholds but continues to
```

Comment:
Can you explain a bit more what "total health" here actually refers to? This is related to my comment in the code itself in which IMO this is not very intuitive. Perhaps actually described that it's normalized by adding up the health of each priority and capping at 100% or something like that?

Reply:
Done.